Content.Client/Audio/DreamSoundEngine.cs

```diff
@@ -0,0 +1,85 @@
+using Content.Client.Resources;
+using Content.Client.Resources.ResourceTypes;
+using Content.Shared.Network.Messages;
+using Robust.Shared.Audio;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Log;
+using Robust.Shared.Network;
+
+namespace Content.Client.Audio {
+    public class DreamSoundEngine : IDreamSoundEngine {
+        private const int SoundChannelLimit = 1024;
+
+        [Dependency] private readonly IDreamResourceManager _resourceManager = default!;
+        [Dependency] private readonly INetManager _netManager = default!;
+
+        private readonly DreamSoundChannel[] _channels = new DreamSoundChannel[SoundChannelLimit];
+
+        public void Initialize() {
+            _netManager.RegisterNetMessage<MsgSound>(RxSound);
+
+            _netManager.Disconnect += DisconnectedFromServer;
+        }
+
+        public void PlaySound(int channel, ResourceSound sound, float volume)
+        {
+            Logger.Debug($"PLAYING SOUND! {sound.ResourcePath}");
```

Comment:
nice debug message

Reply:
oops, forgot to remove them all before opening the PR